apmpackage/apm/changelog.yml

```diff
@@ -1,7 +1,7 @@
 # newer versions go on top
 #
 # change type can be one of: enhancement, bugfix, breaking-change
-- version: "8.4.1"
+- version: "8.4.2"
```

Comment:
**question**: shouldn't this go on top of `8.4.1` ?

Reply:
We haven't actually had any changes for the `8.4.1` package, so we are not releasing it and only bump the version to `8.4.2`.

Reply:
Ah, that entry is a placeholder 🤦